furryfeet_be/furryfeet/models.py

```diff
@@ -8,15 +8,42 @@ class MyModel(models.Model):
 
     def __str__(self):
         return self.name
-    
+
+class DogDetails(models.Model):
+    name = models.CharField(max_length=50)
+    breed = models.CharField(max_length=50)
+    age = models.IntegerField()
+    dog_id = models.CharField(max_length=50)
+    feedbacks = models.TextField()
+    dog_photo = models.URLField(null=True, blank=True)
+
+    def __str__(self):
+        return self.name
+
 class User(AbstractUser):
     first_name = models.CharField(max_length=20)
     last_name = models.CharField(max_length=20)
     email = models.CharField(max_length=200, unique=True)
     password = models.CharField(max_length=20)
     joined_at = models.DateTimeField(auto_now_add=True)
     image = models.URLField(null=True, blank=True)
-    username = None
+    username = models.CharField(max_length=20,blank=True)
     
     USERNAME_FIELD = 'email'
-    REQUIRED_FIELDS = []
\ No newline at end of file
+    REQUIRED_FIELDS = ['username',]
+    dog_details = models.ForeignKey(DogDetails, on_delete=models.CASCADE, null=True, blank=True)
+
+
+class Feedback(models.Model):
```

Comment:
Please sync with @MuratUseini123 on this, otherwise you guys will have merge conflicts